Sources/UI/RequestDetailViewController.swift

```diff
@@ -42,19 +42,30 @@ class RequestDetailViewController: WHBaseViewController {
         super.didReceiveMemoryWarning()
     }
     
-    @objc func shareContent(_ sender: UIBarButtonItem){
+    // MARK: - Actions
+    @objc func openActionSheet(_ sender: UIBarButtonItem){
+        let ac = UIAlertController(title: "Wormholy", message: "Choose an option", preferredStyle: .actionSheet)
+        
+        ac.addAction(UIAlertAction(title: "Share", style: .default) { [weak self] (action) in
+            self?.shareContent(sender)
+        })
+        ac.addAction(UIAlertAction(title: "Share (request as cURL)", style: .default) { [weak self] (action) in
+            self?.shareContent(sender, requestExportOption: .curl)
+        })
+        ac.addAction(UIAlertAction(title: "Share as Postman Collection", style: .default) { [weak self] (action) in
+            self?.shareContent(sender, requestExportOption: .postman)
+        })
+        ac.addAction(UIAlertAction(title: "Close", style: .cancel) { (action) in
+        })
+        if UIDevice.current.userInterfaceIdiom == .pad {
```

Comment:
I think that this check here is not needed.

Reply:
I added the condition at line 60 to handle the correct presentation of the UIAlertController on iPad (now there is a crash when I try to show this UIAlertController on iPad). I will push also a fix for the line 61 because I did a mistake with the assignment.